Add more technical and economical info to JLP and perpetuals guides (#479)
jup-ag/space-station: guides/7-jlp/1-JLP.md

    <meta name="twitter:card" content="summary" />
</head>

> The **Jupiter Liquidity Provider (JLP)** Pool is a liquidity pool that acts as a counterparty to traders on [Jupiter Perps](https://jup.ag/perps). When traders open leveraged positions, they will borrow tokens from the pool.

Author: ```suggestion
The **Jupiter Liquidity Provider (JLP)** Pool is a liquidity pool that acts as a counterparty to traders on [Jupiter Perps](https://jup.ag/perps). Traders borrow tokens from the pool to open leveraged positions on the Jupiter Perpetuals exchange.
```

I think this sounds slightly better in active form, WDYT?

The JLP token derives its value from:

- An index fund of **SOL, ETH, WBTC, USDC, USDT**.
- Trader's **profit and loss**.
- **75%** of the generated fees from **opening** and **closing** fees, **price impact**, **borrowing fees**, and **trading fees** of the pool.

### Jupiter Liquidity Providers (JLPs)

Jupiter Liquidity Providers (JLPs) play a crucial role in the trading ecosystem by supplying the necessary liquidity for perpetual traders. 

> Liquidity providers (LPs) ensure there is sufficient liquidity, allowing traders to test their perpetual trading strategies. As a result, Jupiter can offer its trading service without needing to hold huge amounts of liquidity, and LPs are rewarded with the majority of the fees.

Reviewer: Jupiter Perp match LP and traders

More details on how to get the token in [How To Get JLP](../jlp/How-To-Get-JLP).

:::tip
The best way to purchase or exit JLP is always via [Jupiter Swap](https://jup.ag/swap/USDC-JLP).
:::

 ### Advantages of the JLP System

 The JLP system offers a user-friendly method for participants to earn passive income while contributing to the liquidity and stability of the trading environment:

- LPs do not need to actively "stake" tokens or "harvest" yields - APR earnings are embedded within each JLP token and accumulate automatically (reflected as an increase in the JLP token price).
- The JLP token is also an SPL token, making it easily transferable and tradable like other SPL tokens within the Solana ecosystem.
- AMM pools can be established for trading JLP tokens.

:::info
JLP is a liquidity token for the Jupiter Perpetuals platform, and JUP is the governance token of the JUP DAO.
:::

### Rewards and Earnings

The **APY** of JLP is calculated based on 75% of fees generated from perps trading activities (not including assets appreciation and traders PnL). 

> The generated fees are distributed back to liquidity providers by redepositing the fees into the pool hourly.

Author: ```suggestion
The generated fees are distributed back to liquidity providers
```

We actually distribute fees back to the pool once a week (which is when we calculate the APR of the pool), so we can change that here

> The generated fees are distributed back to liquidity providers by redepositing the fees into the pool weekly (at the same time APY is updated).

Reviewer: fees are deposited into the pool hourly, but APY is calculated weekly based on past week performance

Author: Sorry @wowmiir @hjchan , before this @wowmiir already put hourly but I forgot about the hourly deposit and changed it to weekly instead, so your original post was right @wowmiir

10. **Estimations and Fees:** The user is shown the estimated fee for trading the token they pick and the estimated JLP to be received.

> 11. **JLP Price & Total Supply:** Here, you can find the current `JLP Price` and `Total Supply` displayed. The Price is obtained by dividing `TVL` by `Total Supply`.

Reviewer 2: Good point - added a short blurb below in case they get confused about virtual price

> 11. **JLP Price & Total Supply:** Here, you can find the current `JLP Price` and `Total Supply` displayed. The Price is obtained by dividing `TVL` by `Total Supply`.

Author: Yeah that works!
